cinema/models.py

```diff
@@ -53,18 +53,26 @@ class MovieSession(models.Model):
     show_time = models.DateTimeField()
     movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
     cinema_hall = models.ForeignKey(CinemaHall, on_delete=models.CASCADE)
+    movie = models.ForeignKey(
+        Movie, on_delete=models.CASCADE, related_name="movie_sessions"
+    )
+    cinema_hall = models.ForeignKey(
+        CinemaHall, on_delete=models.CASCADE, related_name="movie_sessions"
+    )
```

Comment:
There are duplicate foreign key definitions for `movie` and `cinema_hall` in the `MovieSession` model. The first set of definitions (lines 54-55) should be removed to avoid redundancy and potential conflicts.